jax/_src/test_util.py

```diff
@@ -493,7 +493,7 @@ def is_cuda_compute_capability_at_least(capability: str) -> bool:
   if not is_device_cuda():
     return False
   d, *_ = jax.local_devices(backend="gpu")
-  return d.compute_capability >= capability
+  return float(d.compute_capability) >= float(capability)
```

Comment:
We should avoid floating point comparisons where we aren't dealing with approximate values. Here I suggest comparing as tuples?

If you form `tuple(int(x) for x in capability.split("."))` then those tuples should compare as you would expect versions to compare (lexicographically).

Reply:
Thanks for the suggestion. Done. PTAL.